main/webapp/modules/core/scripts/index/open-project-ui.js

```diff
@@ -101,15 +101,54 @@ Refine.OpenProjectUI.prototype._fetchProjects = function() {
 };
 
 Refine.OpenProjectUI.prototype._buildTagsAndFetchProjects = function() {
+    this._buildProjectSearchPanel();
     this._buildTagsListPanel();
     this._fetchProjects();
 };
 
+Refine.OpenProjectUI.prototype._buildProjectSearchPanel = function(){
+    var self = this;
+    self._allTags = Refine.TagsManager._getAllProjectTags();
+    var container = self._elmts.projectTags.empty();
+    // Add search menu item
+    var div = $('<div/>')
+    .attr('id','divSearch')
+    .appendTo(container)
+    // Add form to the div on the left
+    var form = $('<div/>')
+    .attr('id','formSearch')
+    .appendTo(div);
+    // Add div for image in the form
+    var divImage = $('<div/>')
+    .attr('id','divImage')
```

Comment:
Maybe a more specific `id` for this `<div>` could be nice (there are other divs with images in this page). For instance `search-icon` (to follow the style of the rest of the DOM ids in this page).